packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/bedrock/integ.invoke-model.ts

```diff
@@ -52,13 +52,32 @@ const prompt2 = new BedrockInvokeModel(stack, 'Prompt2', {
   resultPath: '$',
 });
 
+/** Test for Bedrock Output Path */
 const prompt3 = new BedrockInvokeModel(stack, 'Prompt3', {
   model,
-  inputPath: sfn.JsonPath.stringAt('$.names'),
+  body: sfn.TaskInput.fromObject(
+    {
+      inputText: sfn.JsonPath.format(
+        'Alphabetize this list of first names:\n{}',
+        sfn.JsonPath.stringAt('$.names'),
+      ),
+      textGenerationConfig: {
+        maxTokenCount: 100,
+        temperature: 1,
+      },
+    },
+  ),
   outputPath: sfn.JsonPath.stringAt('$.names'),
 });
 
-const chain = sfn.Chain.start(prompt1).next(prompt2).next(prompt3);
+/** Test for Bedrock s3 URI Path */
+const prompt4 = new BedrockInvokeModel(stack, 'Prompt4', {
+  model,
+  s3InputUri: sfn.JsonPath.stringAt('$.names'),
+  s3OutputUri: sfn.JsonPath.stringAt('$.names'),
+});
+
```

Comment:
Can you run the verification steps listed at the top of this class?
`Stack verification steps`